nautobot/dcim/templates/dcim/module/base.html

```diff
@@ -0,0 +1,97 @@
+{% extends 'generic/object_retrieve.html' %}
+{% load buttons %}
+{% load helpers %}
+
+{% block extra_buttons %}
+        {% if perms.dcim.change_module %}
```

Comment:
I'm guessing this is following a similar pattern to devices, but it's interesting that we gate on this permission here given that adding components doesn't *actually* change the module itself. I also wonder whether the views actually enforce this additional permission or whether it's only a display feature?

More fuel for the v3.0 permissions rework, I imagine.